CMake/absl.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
```

Comment:
Is it possible to make the package relocatable?
This is a good things to have I think and can be achieved by using a combination of `pcfiledir` and `CMAKE_INSTALL_XXXDIR`, e.g.:

```
prefix=${pcfiledir}/../..
...
includedir=${prefix}/${CMAKE_INSTALL_INCLUDEDIR}
```

This feature is discussed in the following links:
- https://wp.colliertech.org/cj/?p=164
- https://bugs.freedesktop.org/show_bug.cgi?id=62018


For information, CMake packages tries to do this by default:
- https://cmake.org/cmake/help/v3.9/manual/cmake-packages.7.html#creating-relocatable-packages

Reply:
Thx. My issue with that is that it breaks some package managers / installer that specify  manually CMAKE_INSTALL_INCLUDEDIR and CMAKE_INSTALL_LIBDIR with absolute path which are not under prefix but in completely different directory. ( Nix does that, https://nixos.org/nix/ ). If you have a solution to go around that, that would be good.

Reply:
Come on, why does nix do this? :(

I wonder if it's the work of the nix recipe to work around this, or if each upstream package need to be non-relocatable to please nix.
The gazebo recipe seems to work around this in its recipe:
- [NixOS/nixpkgs/pkgs/applications/science/robotics/gazebo/default.nix](https://github.com/NixOS/nixpkgs/blob/fd84a79623082280b9d9bbaddd0e580fe42babf4/pkgs/applications/science/robotics/gazebo/default.nix#L34-L36)

Reply:
@Sarcasm One solution to please everybody could be to check if CMAKE_INSTALL_INCLUDEDIR is absolute or not. If it is, just use it, if it is not, we generate relocatable files. It is rather trivial to do.

Reply:
Implemented and fixed. I also re-ordered the libs to avoid any issue when static linking